packages/ui/src/components/_charts/configurable-chart.vue

```diff
@@ -34,8 +35,7 @@ Chart.register(
 )
 
 const props = defineProps<{
-  // TODO 2022-09-29: Type this, this should be generic
-  chartConfiguration: any
+  chartConfiguration: ChartConfiguration
```

Comment:
The comment should stay, because ChartConfiguration is generic and we could type the chartData